.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -29,10 +28,7 @@ write a little note why.
    * `spec_version`: The version of the runtime specification. A full node will not attempt to use its native runtime in substitute for the on-chain Wasm runtime unless all of spec_name, spec_version, and authoring_version are the same between Wasm and native.
    * `impl_version`: The version of the implementation of the specification. Nodes are free to ignore this; it serves only as an indication that the code is different; as long as the other two versions are the same then while the actual code may be different, it is nonetheless required to do the same thing. Non-consensus-breaking optimizations are about the only changes that could be made which would result in only the impl_version changing.
    * `transaction_version`: The version of the extrinsics interface. This number must be updated in the following circumstances: extrinsic parameters (number, order, or types) have been changed; extrinsics or pallets have been removed; or the pallet order in the construct_runtime! macro or extrinsic order in a pallet has been changed. If this number is updated, then the spec_version must also be updated 
-- [ ] If needed, notify the committer this is a draft-release and a tag is needed after merging the PR.
 - [ ] Verify benchmarks & weights have been updated for any modified runtime logics
-- [ ] If needed, bump `version` for every crate.
 - [ ] If import a new pallet, choose a proper module index for it, and allow it in `BaseFilter`. Ensure **every** extrinsic works from front-end. If there's corresponding tool, ensure both work for each other.
 - [ ] If needed, update our Javascript/Typescript APIs. These APIs are offcially used by exchanges or community developers.
-- [ ] If we're going to issue a new release, freeze the code one week early(it depends, but usually it's one week), ensure we have enough time for related testing.
-- [ ] Check if inheriting any upstream runtime storage migrations. If any, perform tests with `try-runtime`.
+- [ ] When integrating upstream changes check if you are inheriting runtime storage migrations. If any, perform tests with `try-runtime`.
```

Comment:
+
- [ ] Search code changes for added/removed StorageItems ( grep for "StorageValue", "StorageMap" in the diffs and find changes to #[pallet::config] sections ), ensure those changes have migrations and test them with `try-runtime`

Reply:
There are more storage items than those two but I get your point.